Desktop/Application/MaxMix/App.xaml.cs

```diff
@@ -40,12 +42,22 @@ void OnDispatcherUnhandledException(object sender, DispatcherUnhandledExceptionE
 
         private void ApplicationStartup(object sender, StartupEventArgs e)
         {
+            var assemblyName = Assembly.GetExecutingAssembly().GetName().Name;
+
+            bool createdNew;
```

Comment:
Should this be renamed to something else like, isOpen, isRunning, isAlreadyRunning, instanceFound?